Formula/p/pdfalyzer.rb

```diff
@@ -0,0 +1,87 @@
+class Pdfalyzer < Formula
+  include Language::Python::Virtualenv
+
+  desc "PDF analysis toolkit"
+  homepage "https://github.com/michelcrypt4d4mus/pdfalyzer"
+  url "https://files.pythonhosted.org/packages/fd/86/d235c3936af7638eed693648d9bf651930dd384aa50d11250e5db9d6b4ab/pdfalyzer-1.14.6.tar.gz"
+  sha256 "d992de52f060559d0ff0086c0fff05fd86219fb8ffe3a48f8c03a7debee98809"
+  license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

https://github.com/michelcrypt4d4mus/pdfalyzer/blob/master/pyproject.toml#L6